lumigator/python/mzai/backend/.alembic/README.md

```diff
@@ -0,0 +1,119 @@
+# Using Alembic with Lumigator
+
+https://alembic.sqlalchemy.org/en/latest/index.html
+
+> [Alembic](https://alembic.sqlalchemy.org/) is a lightweight database migration tool for usage with the
+> [SQLAlchemy](https://www.sqlalchemy.org/) Database Toolkit for Python.
+
+In Lumigator, whenever we make changes to the database (DB) schemas (via our SQLAlchemy models),
+we can create a migration path for the DB, removing blockers for developers working on the codebase.
+
+In future this should be extended to form part of releases so that moving between versions of Lumigator is easy,
+even when the database schemas have breaking changes.
+
+This document covers the ways in which Alembic can be used.
+
+**NOTE**: Alembic creates an additional table in the database which it uses to store the matching revision ID.
+
+## Environment variables
+
+In order to connect to a 'real' database to compare against the models, an `SQLALCHEMY_DATABASE_URL` is required to be
+present in your environment.
+
+You can `export` the variable in your shell, or provide it on each invocation.
+
+e.g.
+
+```bash
+export SQLALCHEMY_DATABASE_URL=sqlite:///local.db
+```
+
+```bash
+SQLALCHEMY_DATABASE_URL=sqlite:///local.db alembic history
+```
+
+Examples shown in this document will presume `SQLALCHEMY_DATABASE_URL` is exported.
+
+If `SQLALCHEMY_DATABASE_URL` is not present then a default of `sqlite:///local.db` will be used (see: `alembic.ini`)
+
+## Viewing migration history
```

Comment:
Non-blocking: might want to mention that alembic will be available in your local environment via virtualenv or you should be able to run with `uv run` directly?

Reply:
@veekaybee does this cover the ways to run things sufficiently?

https://github.com/mozilla-ai/lumigator/pull/307/commits/acf0bd703cd8b5d3b247b0a22cbac8ef202aed56

Reply:
works for me!